packages/accounts-controller/CHANGELOG.md

```diff
@@ -6,6 +6,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [2.0.3]
+### Changed
+- Bump dependency on `@metamask/keyring-controller` to ^8.0.3
```

Comment:
Nit: Since this is also a peer dependency update we can add a little more detail here:

``` suggestion
- Bump dependency and peer dependency on `@metamask/keyring-controller` to ^8.0.3
```